src/extensibility/ExtensionManager.js

```diff
@@ -260,6 +280,18 @@ define(function (require, exports, module) {
             if (!extensions[id]) {
                 extensions[id] = {};
             }
+            
+            if (locationType === LOCATION_USER) {
```

Comment:
This bit of code was injected squarely between the `if (!extensions[id])` above and `extensions[id].installInfo =` below, which should stay together as a pair.  For the reasons given in my previous comment I think this whole piece of added code should just go away... but if you do keep it, please move it down so it's not in the middle of that pair.